modules/rosa-hcp-sts-creating-a-cluster-cli-no-cni-plugin.adoc

```diff
@@ -53,6 +53,11 @@ $ rosa create cluster --hosted-cp --subnet-ids=$SUBNET_IDS --oidc-config-id=$OID
 
 . Check the status of your cluster by running the following command:
 +
+[IMPORTANT]
+====
+When you first log in to the cluster after it reaches `ready` status, the nodes will still be in the `not ready` state until you install your own CNI plugin. After CNI installation, the nodes will change to `ready`. 
+====
```

Comment:
Could this be right after the command block, so that it doesn't interrupt the "by running the following command:"?